docs/src/use/configure/configuration-files.md

```diff
@@ -232,7 +232,7 @@ The `rules` property can do any of the following to extend (or override) the set
 
 ### Using a shareable configuration package
 
-A [sharable configuration](../../developer-guide/shareable-configs) is an npm package that exports a configuration object. Make sure that you have installed the package in your project root directory, so that ESLint can require it.
+A [sharable configuration](../../extend/shareable-configs) is an npm package that exports a configuration object. Make sure that you have installed the package in your project root directory, so that ESLint can require it.
```

Comment:
```suggestion
A [sharable configuration](../../extend/shareable-configs) is an npm package that exports a configuration object. Make sure that you have installed the package in your project root directory so that ESLint can require it.
```

Reply:
see https://github.com/eslint/eslint/pull/16665#discussion_r1058467503